packages/swapper/src/swappers/JupiterSwapper/errorPatterns.ts

```diff
@@ -0,0 +1,25 @@
+export class SolanaLogsError extends Error {
+  constructor(name: string) {
+    super(name)
+    this.name = name
+  }
+}
+
+export enum JUPITER_ERROR_PATTERNS {
+  SLIPPAGE_TOLERANCE_EXCEEDED = 'SlippageToleranceExceeded',
+  UNDER_MINIMUM_AMOUNT = 'RequireGteViolated',
+  CONSUMED_MORE_FEES = 'exceeded CUs meter',
+}
+
+export enum JUPITER_ERROR_NAMES {
+  SLIPPAGE_TOLERANCE_EXCEEDED = 'SlippageToleranceExceeded',
+  UNDER_MINIMUM_AMOUNT = 'UnderMinimumAmount',
+  CONSUMED_MORE_FEES = 'ConsumedMoreFees',
+}
```

Comment:
nitpick: don't pluralize enums

Reply:
Done in [389da60](https://github.com/shapeshift/web/pull/8433/commits/389da60661cd195083f5751a43bb80be8b977aa9)